### PR TITLE
[11.0][FIX] hr_attendance_report_theoretical_time Minor changes

### PR DESCRIPTION
**Pull request by jarroyomorales**
Minor changes I forgot to change in hr_attendance_report_theoretical_time

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why don't use v10 code where I change the menu group for being accessible to everyone?

**Comment by jarroyomorales** [comment]
@pedrobaeza https://github.com/OCA/hr/blob/10.0/hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report_views.xml#L52
Do you mean this?

**Comment by pedrobaeza** [comment]
Exactly

**Comment by jarroyomorales** [comment]
It's the same permission defined by odoo.
https://github.com/odoo/odoo/blob/11.0/addons/hr_attendance/views/hr_attendance_view.xml#L187
If i am not understanding this wrong we want Manual Attendance to have permission, but hr_attendance.group_hr_attendance_user means Officer
Maybe you mean hr_attendance.group_hr_attendance. Is it right?

**Comment by pedrobaeza** [comment]
Well, I mean the equivalent one. From v10 to v11, they have been realigned, but the spirit is the same: to allow attendances users to access to this menu.

File: hr_attendance_report_theoretical_time/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-You need to be at least "Attendance / Manual Attendance" for being able to see
+You need to be at least "Attendance / Officer" for being able to see
```
Comment (rafaelbn) [inline hr_attendance_report_theoretical_time/readme/CONFIGURE.rst]: @jarroyomorales employees must see the report to check if they have done more o less hours

Comment (jarroyomorales) [reply hr_attendance_report_theoretical_time/readme/CONFIGURE.rst]: Okay, I changed it again

**Comment by jarroyomorales** [comment]
Added `active` employees filter by default

**Review by etobella — APPROVED** [review]
LGTM.
Dump version please

**Review by JordiBForgeFlow — APPROVED** [review]
:+1 Code review and tested.

**Pull request merged**